jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -397,11 +398,6 @@ private String debugId() {
         }
       }
 
-      // TODO move this to jaeger-zipkin, this adds tracer tags to zipkin first span in a process
```

Comment:
It's still added, but I've moved that logic into ZipkinSpanConverter as per the TODO